mocha-client-tests/karma.conf.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      // Test dependencies
+      'node_modules/expect.js/index.js',
+      'node_modules/sinon-chrome/bundle/sinon-chrome-webextensions.min.js',
+
+      // Source
+      'addon/*.js',
+
+      // Tests
+      'tests/lib/*.js'
+    ],
+
+    // list of files to exclude
```

Comment:
since this is a little confusing, I would suggest adding a note like "These tests are intended to be run from the add-on itself, not from the karma test suite"